akka-contrib/src/main/scala/akka/contrib/pattern/TransparentExponentialBackoffSupervisor.scala

```diff
@@ -1,203 +0,0 @@
-/**
```

Comment:
Can you have a the movement of this file from:

`akka-contrib/src/main/scala/akka/contrib/pattern/TransparentExponentialBackoffSupervisor.scala` to
`akka-actor/src/main/scala/akka/pattern/BackoffOnRestartSupervisor.scala`

be in a separate commit that comes before this commit (in the same pull request)?
This way we can see the changes between the two versions more easily.